CMakeLists.txt

```diff
@@ -114,14 +114,24 @@ endif(MSVC)
 # Dependencies setup
 ########################################################################
 
-# cpu_features
+# cpu_features - sensible defaults, user settable option
+if(CMAKE_SYSTEM_PROCESSOR MATCHES "(^mips)|(^arm)|(^aarch64)|(x86_64)|(AMD64|amd64)|(^i.86$)|(^powerpc)|(^ppc)")
+option(VOLK_CPU_FEATURES "Volk uses cpu_features" ON)
+else()
+option(VOLK_CPU_FEATURES "Volk uses cpu_features" OFF)
+endif()
+if (VOLK_CPU_FEATURES)
+message(STATUS "Building Volk with cpu_features")
 set(BUILD_PIC ON CACHE BOOL
     "Build cpu_features with Position Independent Code (PIC)."
     FORCE)
 set(BUILD_SHARED_LIBS_SAVED "${BUILD_SHARED_LIBS}")
 set(BUILD_SHARED_LIBS OFF)
 add_subdirectory(cpu_features)
 set(BUILD_SHARED_LIBS "${BUILD_SHARED_LIBS_SAVED}")
+else()
+message(STATUS "Building Volk without cpu_features")
+endif()
```

Comment:
Could you add indentation in this section? This would make it easier to read and understand in the future.